templates/docs/whats-new.md

```diff
@@ -1,10 +1,10 @@
 ---
 wrapper_template: '_layouts/docs.html'
 context:
-  title: Component status
+  title: What's new in Vanilla
 ---
 
-## Component status
+## What's new in Vanilla
```

Comment:
We should bump the headings on this page up one level, currently there's no `h1`, just `h2/3/4`. Also, I wondered whether we need the second "What's new in Vanilla" heading? We could make this heading that, since it's what's listed in the nav

```suggestion
# What's new in Vanilla {{versionMinor}}
```